README.md

```diff
@@ -104,6 +111,21 @@ spec:
           }
         ]
       }
+  secondaryNetwork:
+    multus:
+      image: multus
+      repository: nfvpe
+      version: stable
+      # if config is missing or empty then multus config will be automatically generated from the CNI configuration file of the master plugin (the first file in lexicographical order in cni-conf-dir)
+      config: ''
+#    cniPlugin:
```

Comment:
Ah i see perviously you have intentionally omitted `cniPlugin`.

So id say either remove it like it was before or fully enable it with proper image path and set its status to `ready` below, your choice :)

Reply:
removed